src/main/java/LRUCache.java

```diff
@@ -0,0 +1,68 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@RestController
+@EnableAutoConfiguration
+public class LRUCache {
+
+    private static final Logger log = LoggerFactory.getLogger(Cache.class);
+
+    private Cache cache;
+
+    //Used for integration tests
+    private static int capacity = 5;
+
+    @Autowired
+    public LRUCache() {
+        cache = new Cache(capacity);
```

Comment:
why do you use Autowired and initiate cache by new?

Reply:
I tried to find better way to initiate capacity for integration tests and was trying injection. I forgot to remove Autowired after it didn't work.
I initiate Cache by new here for integration tests.